### ISSUE TITLE
some website elements don't meet recommended contrast standards

### ISSUE THREAD

**Issue by Alys** [HabitRPG/habitica#11448]
EDIT 2021-01-18: Anyone needing greater contrast may like to try @citrusella's [Habitica WCAG AAA Contrast](https://habitica.fandom.com/wiki/Habitica_WCAG_AAA_Contrast) Stylus/Stylish theme.

---

In the Report a Bug guild, Donald Maness / @ddmaness (4d235d28-bd6f-4859-b222-e9a3f45ea212) has reported that the elements below don't meet recommended contrast minimum standards for text and non-text UI elements.

"All Items tested in Chrome on Oct. 16 2019 against WCAG's 2.1 AA standards"
Ref: [1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG21/quickref/?currentsidebar=%23col_overview#contrast-minimum)

I'll leave this as suggestion-discussion for a while and then if there's no objections to this approach I'll mark it as help wanted and suggest that a contributor could claim either all these items at once or just one/some of them. If the latter, I'll keep this post updated with the ones that are still open for claiming. @Tressley might want to provide comments or specific instructions before this is ready to be worked on.

* [x] The grey text in the footer
![image](https://user-images.githubusercontent.com/1495809/67153340-e0f77300-f32a-11e9-8f2e-a59d42b819df.png)

* [ ] The grey text used for completed checklist items in the task menu
![image](https://user-images.githubusercontent.com/1495809/67153350-0be1c700-f32b-11e9-898c-3cd1c914dd17.png)

* [x] The white text used for the donate button in the footer
![image](https://user-images.githubusercontent.com/1495809/67153358-21ef8780-f32b-11e9-8224-40aba10ccd8f.png)

* [ ] The yellow text used for the gold cost in the task section's reward column
![image](https://user-images.githubusercontent.com/1495809/67153360-29169580-f32b-11e9-9669-90ffed34059c.png)

* [x] The blue text used for the mana cost in the skills section
![image](https://user-images.githubusercontent.com/1495809/67153362-2e73e000-f32b-11e9-87fb-5d4738a5aca0.png)

* [ ] The grey text for each column in the task section explaining what the column is used for
![image](https://user-images.githubusercontent.com/1495809/67153363-36cc1b00-f32b-11e9-98b9-6ffe2d9b2396.png)

* [ ] The light blue text used for links in tasks
![image](https://user-images.githubusercontent.com/1495809/67153364-3af83880-f32b-11e9-9fe7-130414017a5c.png)

* [x] The social links in the footer
![image](https://user-images.githubusercontent.com/1495809/67153365-3df32900-f32b-11e9-86fc-909e78830427.png)

* [ ] The white text used for the "daily" and "to do" column counter
![image](https://user-images.githubusercontent.com/1495809/67153367-43507380-f32b-11e9-953f-ae4345aa65e7.png)

**Comment by ddmaness** [comment]
This tool might be helpful in deciding whether/how to tweak contrast ratios https://contrast-ratio.com/?utm_campaign=chrome_series_contrastratio_050417&utm_source=chromedev&utm_medium=yt-desc It provides a quick and easy way to see the current ratio and to check the contrast ratio of proposed changes.

Also note that the contrast recommendation for non-text UI elements (like the social links in the footer) is slightly lower than the recommendation for text contrast.  https://www.w3.org/TR/WCAG21/#non-text-contrast

**Comment by Tressley** [comment]
@ddmaness / @Alys -- Thank you for the feedback! Accessibility concerns are definitely on our radar and I can add these to my list. We've been making some updates recently to meet these standards, starting with tasks, and those should be rolling out soon.

**Comment by Alys** [comment]
I've updated the top post to add this note as a temporary work-around for anyone who may find it helpful: 
EDIT 2021-01-18: Anyone needing greater contrast may like to try @citrusella's [Habitica WCAG AAA Contrast](https://habitica.fandom.com/wiki/Habitica_WCAG_AAA_Contrast) Stylus/Stylish theme.
